app/assets/javascripts/merge_status.coffee

```diff
@@ -42,6 +42,13 @@ class MergeStatusPoller
         container.innerHTML = html
         @onPageChange()
 
+  reloadUrl: =>
+    rollback_checkbox = document.querySelector('input[name="rollbackable"]:checked')
+    if rollback_checkbox == null
+      return window.location.toString()
+    else
+      return window.location.toString() + "&rollbackable=" + rollback_checkbox.value
```

Comment:
The page refresh/poll here gets in the way a little bit, where after "no" is selected, a refresh can kick in which resets the radio buttons. '

This is the workaround, which will stick the current radio button value to the end of the refresh link as a query parameter, and then when we render the html.erb for the page, we will set the default checked radio depending on this param.

I spent way too long banging my head for a good solution to this, and this seems to be the least intrusive (least amount of js radio button manipulation). lmk if anyone has any suggestions

Reply:
one potential solution is: store the state of the `rollbackable` flag in the hash part of the url

Reply:
I think you're describing the behaviour here - I'm storing the state in the url as something like:
`shipit.shopify.io/merge_status?commits=1&rollbackable=true`

Reply:
you could also use `window.localStorage` if you don't want the info in the url. More stuff like expiration to deal with then though.